adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -560,14 +560,16 @@ public void acquireToken(final IWindowComponent fragment, final String resource,
     }
 
     /**
-     * This uses new dialog based prompt. It will create a handler to run the
-     * dialog related code. It will start interactive flow if needed. It checks
-     * the cache to return existing result if not expired. It tries to use
-     * refresh token if available. If it fails to get token with refresh token,
-     * behavior will depend on options. If promptbehavior is AUTO, it will
-     * remove this refresh token from cache and fall back on the UI. Default is
-     * AUTO.
-     *
+     * acquireToken will authorize an end user to call the specified resource. 
+     * The minted access token will be sent to the {@link AuthenticationCallback}
```

Comment:
Note about clarity: "minted" may confuse users into thinking that our library _creates_ access tokens when, in reality, they are given to us.

How about:
- "The access token returned from the STS..."
- "The acquired access token...."
- "The access token"

Reply:
Good call out. Fixed.